tests/test_core.py

```
@@ -1,81 +1,100 @@
     """Tests validation for various IPv4, IPv6, and CIDR formats."""
     assert core.is_valid_ip_or_cidr(address) == expected
 
+
 # --- Test Whitelist Logic ---
 
+
 def test_ip_is_whitelisted(mock_settings):
     """Tests if an IP is correctly identified within a whitelisted CIDR."""
     now = int(time.time())
     whitelist = {"192.168.1.0/24": now + 3600}
     assert core.is_ip_whitelisted("192.168.1.100", whitelist, mock_settings) == True
```

Ruff (0.15.13)

[error] Avoid equality comparisons to `True`; use `...:` for truth checks

Replace comparison

(E712)